docs/project/contribution_tools.md

```diff
@@ -132,7 +132,7 @@ cd carbon-lang
 pre-commit install
 
 # Run tests.
-bazel test //...:all
+bazelisk test //...:all
```

Comment:
```
╚╡which bazel
/opt/homebrew/bin/bazel
╚╡realpath /opt/homebrew/bin/bazel
/opt/homebrew/Cellar/bazelisk/1.22.0/bin/bazelisk
╚╡which bazelisk
/opt/homebrew/bin/bazelisk
╚╡realpath /opt/homebrew/bin/bazelisk
/opt/homebrew/Cellar/bazelisk/1.22.0/bin/bazelisk
```

Reply:
Ah, I just found this reading the docs for "how do i run all tests", and we had previously said to always run bazelisk, not bazel. We could rely on it being a symlink if you like though that might mislead folks who end up using this instruction in other contexts. WDYT?

Reply:
I tend to expect phrasing to be more "install bazelisk to get bazel". If there's something in the docs, can you share a link? We do say above "Consider either aliasing `bazel` to the `run_bazelisk.py` script", similar to how Homebrew is set up. I'm not seeing much reference either way.

I have a preference for `bazel` because it's the tool users should be reading help on, and we'll often be saying bazel instead of bazelisk. Even on this page, there are things like `bazel clean`. Maybe we can get by with just a comment "# Homebrew symlinks bazel to bazelisk." to reduce confusion?

Reply:
Oh maybe I misunderstood a lot, but from [here](https://discord.com/channels/655572317891461132/824137170032787467/1306025859261399084) I understood that I should be running bazelisk, not bazel. On my machine, I downloaded a bazelisk binary and I run that always instead of bazel.

Reply:
Google employees have a pre-installed version of Bazel. However, Carbon is a public project, and we typically try not to write documentation that only applies to Google employees.

Reply:
Right, I guess my understanding of "install bazelisk" is that you get a `bazelisk` binary to run, and it goes and installs/updates bazel, whereas you're saying typically (or maybe just on mac?) it means you get a `bazel` binary to run.

Reply:
Sure, and that's why I suggested a comment above. If you don't think that comment is sufficient, we'd probably need to rewrite all the documentation that talks about running `bazel` to say `bazelisk` instead -- maybe it'd be good to ask for another opinion if you think that's the better path?

Reply:
To be sure, part of my skepticism here is due to the Google issue being Linux-specific, and this change being narrowly applied to MacOS advice. This did get me thinking about other ways to prevent the version skew though, which I think I have in #4612

Reply:
Ok I think maybe this change will make us all happy :) I added a more clear preference to making bazel an alias on linux, and added a comment that it would already be an alias on mac.